cmake/DGtalConfigGenerator.cmake

```
@@ -11,15 +11,13 @@ export(TARGETS  DGtal FILE "${PROJECT_BINARY_DIR}/DGtalLibraryDepends.cmake")
 set(DGTAL_INCLUDE_DIRS "${PROJECT_SOURCE_DIR}/src" "${PROJECT_BINARY_DIR}/src")
 set(DGTAL_LIB_DIR "${PROJECT_BINARY_DIR}/src")
 set(DGTAL_CMAKE_DIR "${PROJECT_BINARY_DIR}")
+set(CMAKE_CXX_STANDARD 11)
 
```

Also add
```suggestion
set(CMAKE_CXX_STANDARD_REQUIRED TRUE)

```
to avoid fallback to previous standard version if C++11 is not available (see https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD_REQUIRED.html#prop_tgt:CXX_STANDARD_REQUIRED)